guetzli.make

```diff
@@ -11,20 +11,28 @@ endif
 .PHONY: clean prebuild prelink
 
 ifeq ($(config),release)
-  RESCOMP = windres
+  ifeq ($(origin CC), default)
```

Comment:
Please regenerate with --os=linux. That generates makefiles that appear to wotk on Mac, but --os=macosx generates makefiles that don't work on a typical Linux.